base/shell/explorer/util.cpp

```diff
@@ -237,3 +237,24 @@ GetVersionInfoString(IN LPCWSTR szFileName,
 
     return bRet;
 }
+
+HRESULT ShellExecuteCommand(PCWSTR Command)
```

Comment:
Maybe mention something like
```suggestion
// See also shlwapi!utils.c:ShellExecuteCommand()
HRESULT ShellExecuteCommand(_In_ PCWSTR Command)
```
Could be useful in the future to know we have a similar function over there, in case one needs to extend one and bring similar code to the other function as well.

Or, could it be possible that explorer could instead invoke one of these ordinal-only shlwapi functions? :thinking:

Reply:
Vista has ShellExecCmdLine but we can't call that from outside shell32.

ShellExec_RunDLL almost fits but the way you pass the flags is such a huge hack and requires me to allocate a new command line. Geoff Chappell claims this function is documented but I can't find it on MSDN...